packages/sdk-themes/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@team-monite/sdk-themes",
+  "version": "0.1.0",
+  "private": true,
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "scripts": {
+    "dev": "yarn clean && tsc --watch",
+    "build": "yarn clean && tsc",
+    "clean": "rm -rf dist"
```

Comment:
Please, add `rimraf`, it's more convenient

Reply:
fixed